browse-everything.gemspec

```diff
@@ -26,10 +26,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'google_drive', '~> 2.1'
   spec.add_dependency 'googleauth', '0.6.6'
   spec.add_dependency 'puma', '~> 3.11'
-  spec.add_dependency 'rails', '>= 4.2', '< 6.0'
+  spec.add_dependency 'rails', '>= 4.2'
   spec.add_dependency 'ruby-box'
   spec.add_dependency 'signet', '~> 0.8'
-  spec.add_dependency 'sprockets', '~> 3.7'
+  spec.add_dependency 'sprockets'
```

Comment:
Sprockets becomes a problem with `engine_cart` unless `--skip-sprockets` is passed to the Rails generator. This needs to be resolved in order to remove `sprockets` as a dependency of the Gem (at least, for Rails 6.y.z releases).